packages/autorest.typescript/test/rlcIntegration/generated/bodyStringRest/samples-dev/stringGetBase64EncodedSample.ts

```diff
@@ -14,14 +14,14 @@ dotenv.config();
  * @summary Get value that is base64 encoded
  * x-ms-original-file: /@microsoft.azure/autorest.testserver/swagger/examples/string_getBase64Encoded.json
  */
-async function stringPutNull() {
+async function stringGetBase64Encoded() {
```

Comment:
Just wondering why we have these samples changed?

Reply:
https://github.com/Azure/autorest.typescript/pull/2032#discussion_r1337101537 see the comment here